src/components/member-profile/contribution/index.js

```diff
@@ -63,11 +66,19 @@ const ContributionCard = ({
   urlObj,
 }) => {
   const {
-    task: { title, startedOn, endsOn, status, purpose },
+    task: { id, title, startedOn, endsOn, status, purpose, isNoteworthy },
     prList,
   } = contribution;
   const isTitleAvailable = !!title;
   const featureTitle = isTitleAvailable ? title : prList[0].title;
+  const [showSettings, setShowSettings] = useState(false);
+  const { isOptionKey, setIsOptionKey } = userContext();
+
+  const handleSettingsButton = () => {
```

Comment:
Nit: Method name can be better
```suggestion
  const revealSettingsButton = () => {
```

Reply:
will do that